src/cargo/ops/cargo_rustc/fingerprint.rs

```diff
@@ -385,7 +375,7 @@ fn calculate<'a, 'cfg>(cx: &mut Context<'a, 'cfg>, unit: &Unit<'a>)
         rustc: util::hash_u64(&cx.config.rustc()?.verbose_version),
         target: util::hash_u64(&unit.target),
         profile: util::hash_u64(&unit.profile),
-        features: format!("{:?}", features),
+        features: format!("{:?}", cx.resolve.features_sorted(unit.pkg.package_id())),
```

Comment:
I think this is the only place where behavior is changed, because we don't print `Some`, `None`. But fingerprints can change between cargo releases, can't they?